### DISCUSSION TITLE
Please add Jupyter Notebook preview feature

### DISCUSSION THREAD

**Discussion by AayushSameerShah** [community/community#4232] [Mobile] [answered]
I use Jupyter notebooks a lot in my repository and I often want to see them in the android app, but instead it shows raw code for it.

Please allow the preview of jupyter notebook in mobile

**Comment by amrehman** [comment, marked as answer]
Hi @AayushSameerShah 
Thanks a lot for getting in touch.
I am sorry to hear that you are having some trouble here; definitely sounds like something our team should work on.

I will create an issue on our side and we will try to get to this asap!

Please keep the feedback coming.

**Reply by mabu-dev** [reply]
What's the status of this request?
